src/style.css

```diff
@@ -0,0 +1,133 @@
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+}
+
+body {
+  margin: 0;
+}
+
+.card {
+  width: 200px;
+  border: 1px solid #f3f3f3;
+
+  border-radius: 5px;
+  display: flex;
+  flex-direction: column;
+
+  padding: 32px 16px 16px;
+}
+
+.card__image {
+  width: 160px;
+  height: 134px;
+  margin-left: 3px;
+}
+
+.head {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+
+  color: #060b35;
+  text-align: left;
+  margin-top: 40px;
+
+  margin-bottom: 0;
+}
+
+.parag {
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+
+  text-align: left;
+  color: #616070;
+  margin-top: 4px;
+  
+  margin-bottom: 16px;
+}
+
+.mark {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-bottom: 24px;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+
+  background-image: url(images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars__star:nth-child(-n+4) {
+  background-image: url("images/star-active.svg");
+}
+
+.reviews {
+  color: #060b35;
+  line-height: 14px;
+  font-weight: 400;
+  font-size: 10px;
+}
+
+.payment {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-bottom: 16px;
+}
+
+.price {
+  font-size: 12px;
+  line-height: 18px;
+  color: #616070;
+}
+
+.amount {
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 18px;
+
+}
+
+.buy {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+
+  background-color: #00acdc;
+  border-radius: 5px;
+  border: 1px solid transparent;
+  list-style: none;
+
+  height: 40px;
+  width: 166px;
+
+  text-decoration: none;
+  text-transform: uppercase;
+  color: #fff;
+
+  font-weight: 700px;
+  font-size: 14px;
+  line-height: 16px;
+}
+
+.buy:hover {
+  background-color: #fff;
+  color: #00acdc;
+  border-color: #00acdc;
+}
```

Comment:
AS per my understanding, the :hover need to be implemented for card class and not the buy?